Dockerfile

```diff
@@ -59,15 +59,18 @@ RUN go get -u google.golang.org/grpc
 
 RUN go get -u github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway
 RUN go get -u github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger
-RUN go get -u github.com/golang/protobuf/protoc-gen-go
+RUN go get -u google.golang.org/protobuf/cmd/protoc-gen-go
```

Comment:
So, based on
```
(1) github.com/golang/protobuf@v1.3.4 is the most recent pre-APIv2 version of APIv1.

(2) github.com/golang/protobuf@v1.4.0 is a version of APIv1 implemented in terms of APIv2. The API is the same, but the underlying implementation is backed by the new one. This version contains functions to convert between the APIv1 and APIv2 proto.Message interfaces to ease the transition between the two.

(3) google.golang.org/protobuf@v1.20.0 is APIv2. This module depends upon github.com/golang/protobuf@v1.4.0, so any program which uses APIv2 will automatically pick a version of APIv1 which integrates with it.
```
from https://blog.golang.org/protobuf-apiv2 the plan is next:
switch to (2) first, then switch to (3), and currently, master on pre-(1) as `protoc-gen-go` plugin is overwritten by the   `github.com/micro/protobuf/protoc-gen-go`(which is 5 years old)